src/cljs/rems/roles.cljs

```diff
@@ -3,12 +3,14 @@
 (defn is-logged-in? [roles]
   (some #{:logged-in} roles))
 
-(defn is-applicant-or-member? [roles]
-  (some #{:applicant :member} roles))
+(defn show-applications? [roles]
+  (some #{:applicant :member :reporter} roles))
 
-(defn is-reviewer? [roles]
-  (some #{:handler :commenter :decider :past-commenter :past-decider :reporter} roles))
+(defn show-all-applications? [roles]
+  (some #{:reporter} roles))
 
-(defn is-admin? [roles]
-  ;; TODO: add admin role (owner is a business user; admin is a technical user)
+(defn show-reviews? [roles]
+  (some #{:handler :commenter :decider :past-commenter :past-decider} roles))
+
+(defn show-admin-pages? [roles]
```

Comment:
Is this now rather permissions.cljs, not roles? Pretty nitpicky. I agree that `is-applicant-or-member` was not really the most descriptive name to the relevant action/permission.

Reply:
At least they still take roles as parameters, so they are to some extent about roles. I'm feeling too lazy to change the namespace name. 😉